agent/ecscni/plugin.go

```diff
@@ -37,6 +39,7 @@ const (
 	currentECSCNIVersion = "2018.10.0"
 	currentECSCNIGitHash = "93f4377604504bff92e7555da73b0cba732a4fbb"
 	currentVPCCNIGitHash = "76973f587f9dfb0b40092a78fb5623c9547bc647"
```

Comment:
can you check why you have update on amazon-vpc-plugins submodule without changing this git hash?

Reply:
Fixed it.